EasyPost.NetFramework/Tracker.cs

```diff
@@ -87,14 +87,14 @@ public static Tracker Create(string carrier, string trackingCode)
         ///     All invalid keys will be ignored.
         /// </param>
         /// <returns>Instance of EasyPost.ShipmentList.</returns>
```

Comment:
update the comment

Reply:
Resolved in [73a3961](https://github.com/EasyPost/easypost-csharp/pull/223/commits/73a39618c0e40963cd3dd1b750ef53a13e56f3d1)